dataprofiler/data_readers/filepath_or_buffer.py

```diff
@@ -0,0 +1,31 @@
+from io import open, StringIO, BytesIO
+
+from . import data_utils
+
+class FileOrBufferHandler():
+    def __init__(self, stream_or_buffer, open_method='r', options=dict()):
+        self._stream_or_buffer = stream_or_buffer
+        self.open_method = open_method
+        self.options = options
```

Comment:
if we are going to have `_file` below, we should add it to the init here as `None` to start